BYVAM0/BYVAM0_tests/Mocks/MockDisplay.cs

```diff
@@ -0,0 +1,38 @@
+﻿using BYVAM0.Interfaces;
+using BYVAM0.Model;
+
+namespace BYVAM0_tests.Mocks
+{
+    public class MockDisplay : IDisplay
```

Comment:
Ez nem mock, ez egy Fake. Különbségek:
Fake - Csak helyettesítő implementáció, ami semmit se csinál
Stub - Helyettesítő részleges implementáció
Spy - Helyettesítő részleges vagy teljes implementáció, ami lehetőséget ad a megfigyelésre is. Pl egy számlálón keresztül, hogy mennyiszer lett egy adott metódus meghívva.
Mock - Dinamikusan generált. Konfigurációtól függően lehet Fake, Stub vagy Spy is akár.